src/structures/ClientUser.js

```diff
@@ -282,33 +282,32 @@ class ClientUser extends User {
     if (options.guild instanceof Guild) options.guild = options.guild.id;
     Util.mergeDefault({ limit: 25, roles: true, everyone: true, guild: null }, options);
 
-    return this.client.api.users('@me').mentions({ query: options }).get()
+    return this.client.api.users('@me').mentions.get({ query: options })
     .then(data => data.map(m => new Message(this.client.channels.get(m.channel_id), m, this.client)));
   }
 
   /**
    * Creates a guild.
    * <warn>This is only available when using a user account.</warn>
    * @param {string} name The name of the guild
-   * @param {string} region The region for the server
+   * @param {string} [region] The region for the server
```

Comment:
I could create guilds with an user account and a bot account with just a name, seems pretty optional to me.